contracts/Whitelist.sol

```diff
@@ -67,4 +93,14 @@ contract Whitelist is Ownable {
 
         whitelistedProduct[_productHash] = true;
     }
+
+    /**
+     * @notice set a collateral address as whitelisted
+     * @param _collateral collateral address
+     */
+    function _setWhitelistedCollateral(address _collateral) internal {
+        require(whitelistedCollateral[_collateral] == false, "Collateral already whitelisted");
```

Comment:
Same suggestion as I give to Haythem, we could include the contract name in the require message to smooth the testing process. (`Collateral already whitelisted ` => `Whitelist: Collateral already whitelisted.`)